core-bundle/contao/templates/twig/backend/search/_container.html.twig

```diff
@@ -0,0 +1,41 @@
+{% trans_default_domain "contao_default" %}
+
+{% set search_container_attributes = attrs()
+    .addClass(['backend-search', 'search--hidden'])
+    .set('data-controller', 'contao--backend-search')
+    .set('data-action', 'keydown.esc@document->contao--backend-search#close click@document->contao--backend-search#documentClick')
+    .set('data-contao--backend-search-min-characters-value', '3')
+    .set('data-contao--backend-search-hidden-class', 'search--hidden')
+    .set('data-contao--backend-search-initial-class', 'search--initial')
+    .set('data-contao--backend-search-loading-class', 'search--loading')
+    .set('data-contao--backend-search-invalid-class', 'search--invalid')
+    .set('data-contao--backend-search-results-class', 'search--results')
+    .set('data-contao--backend-search-route-value', path('contao_backend') ~ '/search?keywords=')
+    .mergeWith(tabs_container_attributes|default)
+%}
+
+{% set search_input_attributes = attrs()
+    .addClass('tl_text')
+    .set('autocomplete', 'off')
+    .set('placeholder', 'MSC.search'|trans)
+    .set('name', 'keywords')
+    .set('data-contao--backend-search-target', 'input')
+    .set('data-action', 'focus->contao--backend-search#open input->contao--backend-search#performSearch')
+%}
+
+<div {{ search_container_attributes }}>
+    <div class="backend-search_overlay"></div>
+    <div class="backend-search_inner">
+        <label for="backend-search-input">
+            <span class="invisible">{{ 'MSC.search'|trans }}</span>
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search"><circle cx="11" cy="11" r="8"/><path d="m21 21-4.3-4.3"/></svg>
```

Comment:
If this wasn't an inline SVG, Webpack would optimize it automatically it and we could provide a dark mode variant.

Reply:
See 5e286c4fbf6bdc39ee35927138676f5b90cc6b83 and df5e5fbc64928e6ee120a9a2617ec768a336d3ae